packages/project-disaster-trail/src/components/Game/TaskScreen/index.js

```diff
@@ -1,40 +1,240 @@
-import React, { memo, useEffect } from "react";
+import { memo, useEffect, useState, Fragment } from "react";
 import { PropTypes } from "prop-types";
 import { connect } from "react-redux";
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
 
 import { goToNextChapter } from "../../../state/chapters";
-import { getActiveTaskData } from "../../../state/tasks";
+import {
+  getActiveTaskData,
+  completeTask,
+  getWeightedTasks
+} from "../../../state/tasks";
+import { getPlayerKitItems } from "../../../state/kit";
+import OrbManager from "../OrbManager";
 import ChooseScreen from "./ChooseScreen";
 import SolveScreen from "./SolveScreen";
+import DurationBar from "../../atoms/DurationBar";
+import Ticker from "../../atoms/Ticker";
+import TaskDebugger from "../../atoms/TaskDebugger";
+import Timer from "../../../utils/timer";
 
-const TaskScreen = ({ activeTask, endChapter }) => {
-  // Prevent calling useEffect again with 2nd argument so timer isn't deleted
+import * as ACTIONS from "../../../constants/actions";
+import usePrevious from "../../../state/hooks/usePrevious";
+
+const mapAndInfoStyle = css`
+  position: relative;
+  display: grid;
+  overflow: hidden;
+  width: 100%;
+  height: 100%;
+  grid-template-columns: 1fr;
+  background: beige;
+`;
+
+const TaskScreen = ({
+  activeTask,
+  completeActiveTask,
+  endChapter,
+  debug = true,
+  playerKitItems,
+  weightedTasks
+}) => {
+  const prevActiveTask = usePrevious(activeTask);
+  const [action, setAction] = useState(ACTIONS.SOLVING);
```

Comment:
Nit-picky, but just for organization I feel like it would be nice to have this by the other `useState` expressions